aspnetcore/fundamentals/startup.md

```diff
@@ -74,15 +74,7 @@ A common use of [dependency injection](xref:fundamentals/dependency-injection) i
 
 [!code-csharp[](startup/sample_snapshot/Startup2.cs?highlight=7-8)]
 
-::: moniker-end
-An alternative to injecting `IWebHostEnvironment` is to use a conventions-based approach.
-::: moniker range=">= aspnetcore-3.0"
-
-::: moniker-end
-
-::: moniker range="< aspnetcore-3.0"
 An alternative to injecting `IHostingEnvironment` is to use a conventions-based approach.
-::: moniker-end
```

Comment:
Need to preserve this `moniker-end` ... looks like the block starts on Line 65.

It also looks like this line (deleted here) ...

> An alternative to injecting `IWebHostEnvironment` is to use a conventions-based approach.

... should go inside the prior (>=3.0) block ... **_BUT_** ... I am assuming that convention-based startup is still a thing. I think it is. I haven't heard otherwise.